nodeup/pkg/model/protokube.go

```diff
@@ -232,16 +232,25 @@ func (t *ProtokubeBuilder) ProtokubeFlags(k8sVersion semver.Version) *ProtokubeF
 
 func (t *ProtokubeBuilder) ProtokubeEnvironmentVariables() string {
 	// TODO temporary code, till vsphere cloud provider gets its own VFS implementation.
-	if fi.CloudProviderID(t.Cluster.Spec.CloudProvider) == fi.CloudProviderVSphere && (os.Getenv("AWS_ACCESS_KEY_ID") != "" || os.Getenv("AWS_SECRET_ACCESS_KEY") != "") {
+	if fi.CloudProviderID(t.Cluster.Spec.CloudProvider) == fi.CloudProviderVSphere &&
+		(os.Getenv("MINIO_ACCESS_KEY_ID") != "" || os.Getenv("MINIO_SECRET_ACCESS_KEY") != "" || os.Getenv("MINIO_REGION") != "" || os.Getenv("MINIO_ENDPOINT") != "") {
 		var buffer bytes.Buffer
 		buffer.WriteString(" ")
```

Comment:
Can this be encapsulated into single function? 
for example: 
AddEnvVar(buffer, MINIO_ACCESS_KEY_ID)
AddEnvVar(buffer, MINIO_SECRET_ACCESS_KEY)